docs/src/modules/components/overview/mainDemo/PickerButton.tsx

```diff
@@ -3,67 +3,67 @@ import dayjs, { Dayjs } from 'dayjs';
 import Button from '@mui/material/Button';
```

Comment:
- Use `DatePickerFieldProps`
- Use `useParsedFormat` to generate the label of the `Button` instead of setting the format in the `label` of the picker (the end result is the same)
- Use `usePickersContext` to open the picker
- Use `useSplitFieldProps` to be able to forward the props to the DOM
- Use `useValidation`